src/core/styles/_constants.scss

```diff
@@ -38,8 +38,8 @@ $color-purple-100: hsl(263deg 55% 83%); // #cebceb
 $color-purple-50: hsl(260deg 47% 96%); // #f4f1fa
 
 // Blue colors
-$color-mentorship-banner: hsl(198deg 83% 70%); // #73ccf2
-$color-mentorship-header: hsl(198deg 86% 55%); // #2BB4EF
+$color-mentorship-100: hsl(198deg 100% 80%); // #9BE1FF
+$color-mentorship-200: hsl(198deg 95% 62%); // #41C3FA
```

Comment:
Let's not name colors by the places where they are used

Reply:
Done